src/server/controllers/back.controller.js

```diff
@@ -1,16 +1,13 @@
-const pathJSON = require("../services/path.json");
 const { moveAlongPath, editPath } = require("../services/path.service");
 
 const backController = (currentPage, previousAnswers) => {
   let previousPage;
+  let newPath;
+  const previousAnswersWithFallback = previousAnswers || {};
```

Comment:
could use default values to do this? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters